src/events.js

```diff
@@ -1,309 +1,292 @@
-const {
-  AnimationEvent,
-  ClipboardEvent,
-  CompositionEvent,
-  DragEvent,
-  Event,
-  FocusEvent,
-  InputEvent,
-  KeyboardEvent,
-  MouseEvent,
-  ProgressEvent,
-  TouchEvent,
-  TransitionEvent,
-  UIEvent,
-  WheelEvent,
-} = typeof window === 'undefined' ? /* istanbul ignore next */ global : window
-
 const eventMap = {
   // Clipboard Events
   copy: {
-    EventType: ClipboardEvent,
+    EventType: 'ClipboardEvent',
```

Comment:
Probably because of changes like this bundler didn't bother to shim things like ClipboardEvent (not that it should as such shimming won't work on custom jsdom instances, just on global window)